pkg/planner/core/exhaust_physical_plans.go

```diff
@@ -3882,9 +3882,9 @@ func exhaustPhysicalPlans4LogicalCTE(lp base.LogicalPlan, prop *property.Physica
 	p := lp.(*logicalop.LogicalCTE)
 	pcte := PhysicalCTE{CTE: p.Cte}.Init(p.SCtx(), p.StatsInfo())
 	if prop.IsFlashProp() {
-		pcte.storageSender = PhysicalExchangeSender{
+		pcte.storageSender = physicalop.PhysicalExchangeSender{
 			ExchangeType: tipb.ExchangeType_Broadcast,
-		}.Init(p.SCtx(), p.StatsInfo())
+		}.Init(p.SCtx(), p.StatsInfo(), 0)
```

Comment:
i think 0 should be the selectOffset, which could be derived from p.QueryBlockOffset()

Reply:
will change it back